Src/Sarif.PatternMatcher.Cli/ExportSearchDefinitionsOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+using CommandLine;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Cli
+{
+    [Verb("export-search-definitions", HelpText = "Export search definitions file from specific formats.")]
+    internal class ExportSearchDefinitionsOptions
```

Comment:
I am unclear about the operation of this file as well. Could you add a comment?

Reply:
Added a summary explaining what it would do.